lib/sipPeer.js

```diff
@@ -53,15 +53,21 @@ SipPeer.prototype.delete = function(callback){
 SipPeer.prototype.getTns = function(number, callback){
   var url = this.client.concatAccountPath(SITE_PATH + "/" + this.siteId + "/" + SIP_PEER_PATH + "/" + this.id + "/" + TNS_PATH);
   var field = null;
-  if(arguments.length === 2){
-    url += "/" + number;
-  }
-  else{
+  var query = null;
+  if(arguments.length === 1){
+    query = {
+      page: 1,
+      size: 50000
```

Comment:
`5000` may be a better default size, I think that's what iris generally uses for TNs

Reply:
done